llvm/test/CodeGen/CHERI-Generic/regenerate-all.py

```diff
@@ -81,6 +81,9 @@ def __init__(self):
         parser = argparse.ArgumentParser()
         parser.add_argument("--llvm-bindir", type=Path, required=True)
         parser.add_argument("--verbose", action="store_true")
+        parser.add_argument("--arch", default='all', 
```

Comment:
See https://git.morello-project.org/morello/llvm-project/-/commit/9ce8358caa77e9d71fe0278b639070bdcbad7ca3 for Morello which looks at the supported targets.